server/middleware/token.js

```diff
@@ -11,6 +11,18 @@ var assert = require('assert');
 
 module.exports = token;
 
+/*
+ * Rewrite the url to replace current user literal with the logged in user id
+ */
+function rewriteUserLiteral(req, currentUserLiteral) {
+  if (req.accessToken && req.accessToken.userId && currentUserLiteral) {
+    // Replace /me/ with /current-user-id/
+    req.url = req.url.replace(
+      new RegExp('/' + currentUserLiteral + '(/|$|\\?)', 'g'),
```

Comment:
This assumes "currentUserLiteral" does not contain any special regexp characters like ".". This requirement should be captured in the docs and API docs.